fmn/rules/utils.py

```diff
@@ -148,10 +148,7 @@ def _get_pkgdb2_packages_for(config, username):
 
     data = req.json()
 
-    packages_of_interest = \
-        data['point of contact'] + \
-        data['co-maintained'] + \
-        data['watch']
+    packages_of_interest = sum([data[flag] for flag in flags], [])
```

Comment:
Yeah!

As its second argument, ``sum()`` takes the object that it should *start* the sum with:

```python
>>> a, b, c = list(range(2)), list(range(3)), list(range(1))
>>> sum([a, b, c], [])
[0, 1, 0, 1, 2, 0]
>>> sum([1, 1, 1], 0)
3
>>> sum([1, 1, 1], 20)
23
```